<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlStandaloneAppWizardSourcesPage</class>
 <widget class="QWizardPage" name="QmlStandaloneAppWizardSourcesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="mainQmlFileGroupBox">
     <property name="title">
      <string>Main Qml file</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="Utils::PathChooser" name="mainQmlFileLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qmlModulesGroupBox">
     <property name="title">
      <string>Qml Modules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="qmlModulesListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
